pkg/plugin/v3/scaffolds/init.go

```diff
@@ -21,16 +21,19 @@ import (
 	"io/ioutil"
 	"path/filepath"
 
+	"sigs.k8s.io/kubebuilder/pkg/plugin/v3/scaffolds/internal/templates/config/hack"
+	"sigs.k8s.io/kubebuilder/pkg/plugin/v3/scaffolds/internal/templates/config/kdefault"
+	"sigs.k8s.io/kubebuilder/pkg/plugin/v3/scaffolds/internal/templates/config/rbac"
```

Comment:
nit: do we want to group these imports with the other `"sigs.k8s.io/kubebuilder/pkg/plugin/v3/scaffolds/internal/templates/*` imports in lines 33-36?